src/Tickets/Emails/Legacy.php

```diff
@@ -0,0 +1,228 @@
+				}
+			} else {
+				break;
+			}
+		}
+
+		return $sent;
+	}
+
+	/**
+	 * Dispatches a confirmation email that acknowledges the user has RSVP'd
+	 * including the tickets.
+	 *
+	 * @since TBD
+	 *
+	 * @param null|boolean   $pre      Previous value from the filter, mostly will be null.
+	 * @param int            $order_id The order ID.
+	 * @param int            $event_id The event ID.
+	 * @param Tickets_Module $module   Commerce module we are using for these emails.
+	 */
+	public function send_rsvp_email( $pre, $order_id, $event_id = null, $module = null ) {
+		if ( ! $module instanceof Tickets_Module ) {
+			return false;
+		}
+
+		$all_attendees = $module->get_attendees_by_order_id( $order_id );
+
+		$to_send = [];
+
+		if ( empty( $all_attendees ) ) {
+			return false;
+		}
+
+		// Look at each attendee and check if a ticket was sent: in each case where a ticket
+		// has not yet been sent we should a) send the ticket out by email and b) record the
+		// fact it was sent.
+		foreach ( $all_attendees as $single_attendee ) {
+			// Do not add those attendees/tickets marked as not attending (note that despite the name
+			// 'qr_ticket_id', this key is not QR code specific, it's simply the attendee post ID).
+			$going_status = get_post_meta( $single_attendee['qr_ticket_id'], $module::ATTENDEE_RSVP_KEY, true );
+			if ( in_array( $going_status, $module->get_statuses_by_action( 'count_not_going' ), true ) ) {
+				continue;
+			}
+
+			// Only add those attendees/tickets that haven't already been sent.
+			if ( empty( $single_attendee['ticket_sent'] ) ) {
+				$to_send[] = $single_attendee;
+			}
+		}
+
+		/**
+		 * Controls the list of tickets which will be emailed out.
+		 *
+		 * @param array $to_send       list of tickets to be sent out by email
+		 * @param array $all_attendees list of all attendees/tickets, including those already sent out
+		 * @param int   $order_id
+		 */
+		$to_send = (array) apply_filters( 'tribe_tickets_rsvp_tickets_to_send', $to_send, $all_attendees, $order_id );
+
+		if ( empty( $to_send ) ) {
+			return false;
+		}
+
+		// For now all ticket holders in an order share the same email.
+		$to = $all_attendees['0']['holder_email'];
+
+		if ( ! is_email( $to ) ) {
+			return false;
+		}
+
+		$email_class = tribe( Email\RSVP::class );
+
+		if ( ! $email_class->is_enabled() ) {
+			return false;
+		}
+
+		$use_ticket_email = tribe_get_option( $email_class->get_option_key( 'use-ticket-email' ), false );
+		if ( ! empty( $use_ticket_email ) ) {
+			$email_class = tribe( Email\Ticket::class );
+		}
+
+		$email_class->set( 'post_id', $event_id );
+		$email_class->set( 'tickets', $all_attendees );
+		$subject     = $email_class->get_subject();
+		$content     = $email_class->get_content( [ 'tickets' => $all_attendees ] );
+		$headers     = $email_class->get_headers();
+		$attachments = $email_class->get_attachments();
+
+		$sent = tribe( Email_Sender::class )->send( $to, $subject, $content, $headers, $attachments );
```

Comment:
This should be replaced with:

```
$email_class->set( 'post_id', $event_id );
$email_class->set( 'tickets', $all_attendees );
$email_class->recipient = $to;

$sent = $email_class->send();
```

Reply:
Added a `@todo` so I can handle that afterwards.